src/fragments/lib/auth/native_common/getting_started/common.mdx

```diff
@@ -57,6 +66,88 @@ import flutter5 from "/src/fragments/lib/auth/flutter/getting_started/12_amplify
 
 <Fragments fragments={{flutter: flutter5}} />
 
+</Block>
+<Block name="Existing resources">
+Existing Authentication resources from AWS (e.g. Amazon Cognito UserPools or Identity Pools) can be used with the Amplify Libraries by running:
+
+```bash
+amplify import auth
+```
+
+For more details, see how to [Use an existing Cognito User Pool and Identity Pool](/cli/auth/import).
+
+If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can be used by referencing them in your `amplifyconfiguration.json` file.
```

Comment:
```suggestion
If you are not using the Amplify CLI, a Cognito User Pool and Identity Pool can be used by referencing them in your `amplifyconfiguration.json` or `amplifyconfiguration.dart` file.
```

In amplify-flutter, the file would be `amplifyconfiguration.dart`. The suggestion is from this [reference](https://docs.amplify.aws/cli/start/workflows/#amplify-pull).